action.yml

```diff
@@ -56,11 +56,11 @@ runs:
         test -n "${{inputs.release-profile}}" && echo 'RELEASE_PROFILE=-P${{inputs.release-profile}}' >> $GITHUB_ENV
         cp -v ${{ github.action_path }}/resources/settings.xml $HOME/.m2/
       shell: bash
-    - name: Run maven
-      run: |-
-        test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
-        mvn -B ${{ inputs.maven-additional-options }} package ${SKIP_TESTS}
-      shell: bash
+   - name: Run maven
+    run: |-
+      test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
+      mvn -B ${{ inputs.maven-additional-options }} ${{ inputs.maven-build-options }} package ${SKIP_TESTS}
```

Comment:
@celanthe you need to define the "maven-build-options" input like the other inputs (e.g. maven-release-options). otherwise the action will not accept this input.

Reply:
@celanthe:

    maven-build-options:
      description: Any extra Maven options for the initial build process
      required: false
      default: ""

in the `inputs` section will do it.

Reply:
@gclaussn @VonDerBeck I went ahead and added that to the `inputs` section. Fingers crossed that should work!